### PR TITLE
Render code in markdown docs using the atlaskit code component

### PR DESCRIPTION
**Pull request by dpisani-atl**

### CODE REVIEW

File: packages/website/components/mdx/code/inline.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import { AkCode } from '@atlaskit/code';
+
+interface Props {
+  children: string;
+}
```
Comment (pyupew) [inline packages/website/components/mdx/code/inline.tsx]: Should inline examples not also have language support?

Comment (dpisani-atl) [reply packages/website/components/mdx/code/inline.tsx]: Unfortunately there is no way to specify a language in the markdown for an inline component :(

File: yarn.lock [hunk 2]
```diff
@@ -481,6 +491,15 @@
     exenv "^1.2.2"
     prop-types "^15.5.10"
 
+"@atlaskit/theme@^8.0.0":
```
Comment (Noviny) [inline yarn.lock]: I understand that this is Just How Yarn Works, but I'm so sad that we are already getting two versions of theme (at least!) in our project. Won't somebody think of the resolved dependency deduping in yarn?

(This is me kvetching, not an ask for a solution)

File: packages/website/components/page-templates/package-example.js [hunk 3]
```diff
@@ -86,13 +86,9 @@ const PackageExample = ({ data, fileContents, children }) => {
       <PageContent>
         <ExampleStyle>
           <Header>
-            <h1>Dynamic Table</h1>
+            <h1>{data.pageTitle}</h1>
```
Comment (Noviny) [inline packages/website/components/page-templates/package-example.js]: 🎉

File: packages/website/components/mdx/code/inline.tsx (hunk 1 above)
Comment (Noviny) [reply packages/website/components/mdx/code/inline.tsx]: Sad truths be sad.

File: .eslintrc [hunk 4]
```diff
@@ -25,30 +25,37 @@
       "never",
       { "css": "ignorePackages", "json": "always" }
     ],
-    "react/sort-comp": "off"
+    "react/sort-comp": "off",
+    "import/no-extraneous-dependencies": [
```
Comment (Noviny) [inline .eslintrc]: What's this do?

File: .eslintrc [hunk 5]
```diff
@@ -25,30 +25,37 @@
...
-            "files": ["**/**/*.ts", "**/**/*.tsx"],
-            "parser": "@typescript-eslint/parser",
-            "plugins": ["emotion", "@typescript-eslint"],
-            "rules": {
-                "no-unused-vars": "off",
-                "@typescript-eslint/no-unused-vars": "error"
-            }
-        }
-    ]
+    {
```
Comment (Noviny) [inline .eslintrc]: Curious why this bit got diffed? Different prettier setting somewhere?

File: .eslintrc (hunk 4 above)
Comment (dpisani-atl) [reply .eslintrc]: it lets you import a dev dependency within a test

**Pull request merged**